### PR TITLE
Make safe-string compliant

### PR DESCRIPTION
**Pull request by mseri**
And drop use of bigbuffer

### CODE REVIEW

File: http-svr/http.ml [hunk 1]
```diff
@@ -88,8 +88,12 @@ module Hdr = struct
 end
 
 let output_http fd headers =
-  Unixext.really_write_string fd
-    (String.concat "" (List.map (fun x -> x ^ "\r\n") headers))
+  let output = 
+    List.map (fun x ->  Printf.sprintf "%s\r\n" x |> Bytes.of_string) headers
```
Comment (lindig) [inline http-svr/http.ml]:
```
headers
|> List.map ..
|> Bytes.concat ..
|> fun output -> Unix.write fd output (Bytes.length output)
|> ignore
```

**Comment by mseri** [comment]
I have pushed a fixup, as this branch was based on the first port of stdext to safe-string that was backward incompatible. The current port requires many less changes

**Comment by mseri** [comment]
Unfortunately Travis will not work until we find a way to properly fix `xenctrl.master`

**Comment by mseri** [comment]
Build tested here: https://travis-ci.org/mseri/xs-opam/builds/353816211

**Pull request merged**